pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -500,6 +500,11 @@ func (cm *containerManagerImpl) GetNodeConfig() NodeConfig {
 	return cm.NodeConfig
 }
 
+// GetCgroupRoot returns the cgroup containing all pods
+func (cm *containerManagerImpl) GetCgroupRoot() string {
```

Comment:
`CgroupRoot` doesn't seem to be very descriptive - can't tell from its name that this is the cgroup for **pods**, but if this is a well-known term, then nvm.